app/models/ride.rb

```diff
@@ -1,2 +1,3 @@
 class Ride < ActiveRecord::Base
+  validates_presence_of :origin, :destination, :seat_count, :date, :departure_time
```

Comment:
Line is too long. [82/80]<br>Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.